sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -390,6 +375,10 @@ public AutoConfiguredOpenTelemetrySdk build() {
               .setPropagators(propagators);
 
       openTelemetrySdk = sdkBuilder.build();
+
+      if (registerShutdownHook) {
+        Runtime.getRuntime().addShutdownHook(new Thread(openTelemetrySdk::shutdown));
```

Comment:
Shouldn't that be `close` instead of `shutdown`?
```suggestion
        Runtime.getRuntime().addShutdownHook(new Thread(openTelemetrySdk::close));
```